code/lib/cli/package.json

```diff
@@ -187,6 +187,12 @@
       "types": "./core/babel/index.d.ts",
       "import": "./core/babel/index.js",
       "require": "./core/babel/index.cjs"
+    },
+    "./internal/manager/globals-runtime": {
+      "import": "./core/manager/globals-runtime.js"
+    },
```

Comment:
Where is this one used?

Reply:
It's not used, and only there for symmetry with the preview runtime.